linkernetworks/azure-intelligent-edge-patterns
# Backend/peter dev

## Purpose

* Delete tags and untagged  images before training

## Does this introduce a breaking change?

```
[ ] Yes
[v] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[v] Bugfix
[ ] Feature
[v] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
./manage.py migrate
./manage.py runserver
```

* Test the code

- Create location, cameras and several parts (e.g. part1, part2, part3)
- Configure and train with **part1, part2**
- **Observer Custom Vision and check the followings**
    - **part1 + part2**
    - 2 * 15 = 30+ images
    - 0 untagged images
- Configure and train with **part1, part3**
- **Observer Custom Vision and check the followings**
    - **part1 + part3**
    - 2 * 15 = 30+ images
    - 0 untagged images
    - **part2 been removed**
    - **part2 images been removed**

## What to Check
Verify that the following are valid
* at test the code part

## Other Information
